src/components/policy/policy_regular/src/cache_manager.cc

```diff
@@ -585,6 +585,13 @@ bool CacheManager::SecondsBetweenRetries(std::vector<int>& seconds) {
 
   const std::size_t size =
       pt_->policy_table.module_config.seconds_between_retries.size();
+
+  const std::size_t max_seconds_between_retries_size = 5u;
```

Comment:
@tkireva, so you want another const that will be assigned to that const? what the point?